src/AdminConsole/appsettings.json

```diff
@@ -36,7 +38,30 @@
         }
       },
       "Pro": {
-        "PriceId": "price_1OAdUaBFpfsE1ibivZEYEhaU",
+        "Order": 20001,
+        "PriceId": "price_1N1qT7BFpfsE1ibimVpCIgUe",
+        "Features": {
+          "EventLoggingIsEnabled": false,
+          "EventLoggingRetentionPeriod": 0,
+          "MaxAdmins": 1000,
+          "MaxApplications": 1000
+        },
+        "Ui": {
+          "Label": "Pro",
+          "Price": "$0.05",
+          "PriceHint": "> 10k users, onwards $0.01",
+          "Features": [
+            "All Free features",
+            "Unlimited apps",
+            "Unlimited console administrators",
+            "Dedicated SLA",
+            "Prioritized support"
+          ]
+        }
+      },
+      "Pro-1": {
```

Comment:
nitpick: Any reason for using this SKU instead of including the year? e.g. `PRO-2023-2`